generator/src/main/java/com/scottlogic/deg/generator/restrictions/DateTimeRestrictionsMerger.java

```diff
@@ -5,20 +5,21 @@
         MIN, MAX
     }
 
-    public DateTimeRestrictions merge(DateTimeRestrictions left, DateTimeRestrictions right) {
+    public MergeResult<DateTimeRestrictions> merge(DateTimeRestrictions left, DateTimeRestrictions right) {
```

Comment:
Was this change intentional? There is no difference in the implementation other than the return type.

Reply:
Good spot. Will revert. This was related to some changes whether we should emit values or not for dates of invalid ranges which have been moved.

Reply:
done.